Project/Source Files/Screens/Recipies/RecipiesTableViewCell.swift

```diff
@@ -10,11 +10,22 @@ class RecipiesTableViewCell: UITableViewCell {
     @IBOutlet var recipeTitle: UILabel!
     @IBOutlet var recipeIngredients: UILabel!
 
-    func setupCell(with image: UIImage, title: String, ingredients: String) {
+    func setup(with image: UIImage, title: String, ingredients: String) {
```

Comment:
Please add documentation to the method :) You can find an example in APIClient class.